apps/fluent-tester/src/FluentTester/testPages.win32.ts

```diff
@@ -110,6 +111,11 @@ export const tests: TestDescription[] = [
     component: MenuButtonTest,
     testPage: HOMEPAGE_MENU_BUTTON,
   },
+  {
```

Comment:
E2E testing is incomplete, there's no spec document.

Please see: https://github.com/microsoft/fluentui-react-native/tree/master/apps/fluent-tester/src/E2E#authoring-e2e-test